packages/reactive-element/CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- [Breaking] The `@internalProperty` decorator has been renamed to `@state`.
+- [Breaking] `UpdatingElement` has been renamed to `ReactiveElement` and the `updating-element` package has been renamed to `reactive-element`.
```

Comment:
add a "(since 3.0.0-pre1)" after the `[Breaking]`, and maybe split this into two items.

We should also consider re-exporting ReactiveElement as UpdatingElement from the lit-element package for back-compat.